.eslintrc.js

```diff
@@ -5,9 +5,6 @@ const path = require("path");
 const pkgAliases = fs
     .readdirSync(path.join(__dirname, "packages"))
     .map((pkgName) => {
-        if (pkgName === "kas") {
```

Comment:
Great. I like that we are keeping our configs largely the same for all packages!